server/app/Http/Controllers/GroupController.php

```diff
@@ -37,6 +38,11 @@ public function store(StoreGroupRequest $request)
         $group->status = Group::STATUS_PENDING;
         $group->save();
         $group->addMember(Auth::user()->id, Group::ROLE_OWNER);
+
+        if ($group->individual) {
+            $group->user = Auth::user();
```

Comment:
We could do away with the `user` property on a group as it doesn't make much sense anymore. The owner of the group is a member with the owner role. Not a big deal, I don't think we use it anywhere else, but this could be a good time to just drop it altogether instead of making more use of it.